src/pages/messages/components/NviCandidatesList.tsx

```diff
@@ -56,51 +57,57 @@ export const NviCandidatesList = ({
+        <SearchForm placeholder={t('tasks.search_placeholder')} />
 
-        <CuratorSelector
-          selectedUsername={searchParams.get(NviCandidatesSearchParam.Assignee)}
-          onChange={(curator) => {
-            if (curator) {
-              searchParams.set(NviCandidatesSearchParam.Assignee, curator.username);
+        <Box sx={{ display: 'flex', gap: '1rem', flexWrap: 'wrap' }}>
+          <CuratorSelector
+            selectedUsername={searchParams.get(NviCandidatesSearchParam.Assignee)}
+            onChange={(curator) => {
+              if (curator) {
+                searchParams.set(NviCandidatesSearchParam.Assignee, curator.username);
 
-              const currentStatusFilter = searchParams.get(
-                NviCandidatesSearchParam.Filter
-              ) as NviCandidateSearchStatus | null;
-              if (
-                !currentStatusFilter ||
-                currentStatusFilter === 'pending' ||
-                currentStatusFilter === 'pendingCollaboration'
-              ) {
-                searchParams.set(NviCandidatesSearchParam.Filter, 'assigned' satisfies NviCandidateSearchStatus);
+                const currentStatusFilter = searchParams.get(
+                  NviCandidatesSearchParam.Filter
+                ) as NviCandidateSearchStatus | null;
+                if (
+                  !currentStatusFilter ||
+                  currentStatusFilter === 'pending' ||
+                  currentStatusFilter === 'pendingCollaboration'
+                ) {
+                  searchParams.set(NviCandidatesSearchParam.Filter, 'assigned' satisfies NviCandidateSearchStatus);
+                }
+              } else {
+                searchParams.delete(NviCandidatesSearchParam.Assignee);
               }
-            } else {
-              searchParams.delete(NviCandidatesSearchParam.Assignee);
-            }
-            history.push({ search: searchParams.toString() });
-          }}
-          roleFilter={[RoleName.NviCurator]}
-          sx={{ maxWidth: '20rem' }}
-        />
-        <Select
-          data-testid={dataTestId.tasksPage.nvi.yearSelect}
-          size="small"
-          inputProps={{ 'aria-label': t('common.year') }}
-          value={nviYearFilter}
-          onChange={(event) => {
-            searchParams.set(NviCandidatesSearchParam.Year, event.target.value.toString());
-            history.push({ search: searchParams.toString() });
-          }}>
-          {nviYearFilterValues.map((year) => (
-            <MenuItem key={year} value={year}>
-              {year}
-            </MenuItem>
-          ))}
-        </Select>
+              history.push({ search: searchParams.toString() });
+            }}
+            roleFilter={[RoleName.NviCurator]}
+            sx={{ flex: '0 20rem' }}
+          />
+
+          <AreaOfResponsibilitySelector sx={{ flex: '0 20rem' }} paramName={NviCandidatesSearchParam.Affiliations} />
```

Comment:
Blir litt rart på noen skjermstørrelser, men har ikke funnet noen bedre løsning
![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/66143563-e2bf-423f-8053-f780aed6e950)

Reply:
Mulig å kjøre linje nr 2 fullWidth?

Reply:
At årstallet tar hele bredden tenker du? Ser for meg at det vil bli litt rart også, men vet heller ikke hvordan jeg skal få til det, for man vet vel ikke når den wrappes over til neste linje 🤔

Reply:
Nei, tenkte at kurator og ansvarsområde tar hele bredden. Da ser det ikke ut som årstallet har falt ned awkwardly.

Reply:
`Grid item sm=6 md=4` på kurator og ansvarområde, og `Grid item sm=12 md=4 flex-end` på årstall kunne funket (Veeeldig pseido-kode)

Reply:
Sånn ja, kan se litt på det!

Reply:
La inn `flexGrow`, sånn at de tar den plassen de har. Fører til at filtrene tar en del plass for store skjermer men tenker det er OK. Skal inn flere filter her også over tid, så problemet vil trolig bli mindre etter hvert